cloudflare-template.sh

```diff
@@ -6,27 +6,36 @@ zone_identifier=""                                 # Can be found in the "Overvi
 record_name=""                                     # Which record you want to be synced
 proxy=true                                         # Set the proxy to true or false 
 
+
+###########################################
+## Define date time stamp function
+###########################################
+function ds() {
+            date +"[%F %T]"
```

Comment:
Hi, thanks for your contribution. Would it be possible to keep the white space consistent?

Reply:
Sorry, how do you mean?

Reply:
I think I understood. I changed the tab in the function definition to 2 spaces to match the indentation you were using in your if statements.

Reply:
The indentation on line 14, would it be possible to keep it at 2, like the rest of the script?

Reply:
Got it. I just committed a change that did this.